### PR TITLE
Add Ashimdevnath

### PR DESCRIPTION
**Pull request by Ashimdevnath**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2130"></a>

### CODE REVIEW

File: public/data/Ashimdevnath.json [hunk 1]
```diff
@@ -13,6 +13,11 @@
       "name": "Follow me on Twitter",
       "url": "https://twitter.com/AshimDevnath4",
       "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Linkdin",
+      "url": "https://www.linkedin.com/in/ashim-devnath",
```
Comment (AbbeyIT) [inline public/data/Ashimdevnath.json:19]: Page doesn't exist on my side.

**Review by AbbeyIT — CHANGES_REQUESTED** [review]

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Please resolve the merge conflicts here. Let us know if you need any help with this! :)
Maybe Syncing your fork with the `main` will work, since your branch is `175 commits` behind the base branch!

**Comment by Ashimdevnath** [comment]
Can you help me out what is wrong with thi PR

**Comment by kumarsonsoff3** [comment]
> Can you help me out what is wrong with thi PR

We've recently deployed the `LinkFree V2`, and since your PR is `175 commits` behind the base branch, this maybe causing the conflicts! Try Syncing your `fork` with the `main` branch!

**Comment by Ashimdevnath** [comment]
But This PR is Syncing with Main branch only

**Comment by kumarsonsoff3** [comment]
> But This PR is Syncing with Main branch only

Merge conflicts are arising because the file you're changing is no longer exist in that location in [base branch](https://github.com/EddieHubCommunity/LinkFree). 
Since `Version: 2` is deployed, your file location is changed to [`data/Ashimdevnath.json`](https://github.com/EddieHubCommunity/LinkFree/blob/main/data/Ashimdevnath.json). 

So now, I suggest you to 
- Go to your [`Fork`](https://github.com/Ashimdevnath/LinkFree).
- Sync it by discarding the `2 commits`. 
IMG_20221106_160223
- Then, make the changes in you file `data/Ashimdevnath.json`. 
- Then we're good to goo! 🎉

**Pull request closed without merging**